diff --git a/config.def.h b/config.def.h
index 1c0b587..432c820 100644
--- a/config.def.h
+++ b/config.def.h
@@ -7,11 +7,18 @@ static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
+/* BG color */
 static const char col_gray1[]       = "#222222";
+/* inactive window border color */
 static const char col_gray2[]       = "#444444";
+/* text color */
 static const char col_gray3[]       = "#bbbbbb";
+/* current tag & window text color */
 static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+/* top bar second color and active window border color */
+/* default blue is #005577 */
+/* changed to purple (#9d5fae) by Lukas */
+static const char col_cyan[]        = "#9d5fae";
 static const char *colors[][3]      = {
        /*               fg         bg         border   */
        [SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
