diff --git a/config.def.h b/config.def.h
index 485f9ea..bba54bc 100644
--- a/config.def.h
+++ b/config.def.h
@@ -68,12 +68,12 @@ static const Layout layouts[] = {
 	{ MODKEY|ControlMask|ShiftMask, KEY,      toggletag,      {.ui = 1 << TAG} },
 
 /* helper for spawning shell commands in the pre dwm-5.0 fashion */
-#define SHCMD(cmd) { .v = (const char*[]){ "/usr/local/bin/st", "-e", cmd, NULL } }
+#define SHCMD(cmd) { .v = (const char*[]){ "/usr/bin/urxvt", "-e", cmd, NULL } }
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbordercolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *termcmd[]  = { "urxvt", NULL };
 static const char *firefoxcmd[]  = { "firefox", NULL };
 static const char *flameshotcmd[]  = { "flameshot", "gui", NULL };
 
